waves/src/App.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import "./App.css";
+
+function App() {
+    const [welcome, setWelcome] = useState<String>("Not welcome yet");
+    const [rate, setRate] = useState<String>("No rate received yet");
+
+    import("./wallet-lib/pkg").then(({ hello }) => {
+        let welcome = hello("World");
+        setWelcome(welcome);
+    });
+
+    fetch("http://localhost:3030/rate")
```

Comment:
In prod, this should simply be:

```suggestion
    fetch("/rate")
```

Reply:
`webpack-dev-server` supports a proxy: https://webpack.js.org/configuration/dev-server/#devserverproxy

Means you can tell it to re-write certain requests and point them against some sub-server then you don't have to poison the prod code with conditionals :)

Reply:
I had a look at this but couldn't get it working at the first attempt so I stopped.